src/julia/modules/Zygote/ZygoteBA.jl

```diff
@@ -85,7 +85,9 @@ function compute_ba_J(cams, X, w, obs, feats)
         compute_reproj_err_d_i = x -> compute_reproj_err_d(x, feats[:, i])
         camIdx =  obs[1, i]
         ptIdx = obs[2, i]
-        _, J = Zygote.forward_jacobian(compute_reproj_err_d_i, pack(cams[:, camIdx], X[:, ptIdx], w[i]))
+        y, back = Zygote.forward(compute_reproj_err_d_i, pack(cams[:, camIdx], X[:, ptIdx], w[i]))
```

Comment:
Is this reverse mode?

Reply:
Yes. This call produces a function `back` that computes the derivative for a given direction at the point specified int the `forward` call.
https://fluxml.ai/Zygote.jl/stable/adjoints/

Reply:
Thanks, sorry for the noise.

Reply:
Yes, the terminology is rather unfortunate.  There's "forward mode AD" and "reverse mode AD forward pass".